src/main/java/com/puppycrawl/tools/checkstyle/Main.java

```diff
@@ -75,114 +75,24 @@
      * message in the "messages.properties" file.
      */
     public static final String CREATE_LISTENER_EXCEPTION = "Main.createListener";
+
+    /** Name for the moduleConfig attribute 'tabWidth'. */
+    private static final String ATTRIB_TAB_WIDTH_NAME = "tabWidth";
```

Comment:
please move this constant to CliOptions class, to let this class own all about cli.

Reply:
Done: moved ATTRIB_TAB_WIDTH_NAME to CliOptions.